database/ServiceMongoDb.php

```diff
@@ -114,14 +114,31 @@ public function create()
         }
     }
 
-    public function delete($id)
+    public function update($id)
     {
-        // TODO: Implement delete() method.
+        try{
+            $bulk = new BulkWrite();
+
+            $bulk->update(['_id' => intval($id)], ['$set' =>
```

Comment:
what would happen if you wanted to edit only the address and leave other fields empty? :)